client/app/intake/components/issueModificationRow.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import COPY from '../../../COPY';
+import IssueModificationList from 'app/intake/components/IssueModificationList';
+
+const issueModificationRow = (
+  {
+    fieldTitle,
+    modificationIssueRequestsObj
+  }) => {
+  const sections = [];
+  const modificationIssueRequestsLength = Object.keys(modificationIssueRequestsObj).length - 1;
+  const modificationIssueRequests = Object.entries(modificationIssueRequestsObj).sort();
+
+  let index = 0;
+
+  for (const [key, value] of modificationIssueRequests) {
```

Comment:
```suggestion
  for (const [i, [key, value]] of modificationIssueRequests.entries()) {
```

could do this and not have to manually track the index. Not a dealbreaker for me